tests/functionnal/Item_Rack.php

```diff
@@ -187,7 +187,8 @@ public function testAdd() {
             'name'         => 'Test rack',
             'number_units' => 10,
             'dcrooms_id'   => 0,
-            'position'     => 0
+            'position'     => 0,
+            'entities_id'  => 0,
```

Comment:
I added this to fix tests.

Reply:
I really do not undesrand why tests did not fail before!

Reply:
OK, the check is added on checkUnicity method... So this is normal